BetaUtils.DotNetFramework.Tests/Configuration/WindowsConfigurationProviderTests.cs

```diff
@@ -383,6 +383,20 @@ public void GetStringValue_ValueCanBeParsed_StringValueReturned()
             Assert.AreEqual(expectedValue, resultValue);
         }
 
+        [DataRow(null)]
+        [DataRow("")]
+        [DataTestMethod]
+        public void GetStringValue_WhenKeyIsNullOrEmpty_WillThrowArgumentNullException(string keyValue)
+        {
+            //Arrange
+            var context = new WindowsConfigurationProviderTestContext();
+
+            WindowsConfigurationProvider configurationRepository = context.CreateCoreConfigurationProvider();
+
+            //Act Assert
```

Comment:
These comments in this type of tests do not provide any value as long as tests are kept simple and short, which is not the always the case. Lets take example of marked test: 
` Assert.ThrowsException<ArgumentException>(() => configurationRepository.GetStringValue(keyValue)); `
this line of code is doing two functions, act and assert which in this case complexity is still low since implementation is exemplary.
In real case scenario multiple mock calls would be made to set up the test where readability of the test would drop drastically for example:
//Arrange
//Act Assert
//Arrange
//Act
//Assert

Overall for me its mostly the habit of keeping tests somewhat readable since its easier to troubleshoot the issue in code. 
What's your oppinion?